backend/src/main/java/com/revature/happyfarmersmarket/interceptor/AuthenticationInterceptor.java

```diff
@@ -66,4 +82,13 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         } else response.sendError(401, "Invalid authorization token");
         return false;
     }
+
+
+    private String getTokenFromRequest(HttpServletRequest request) {
```

Comment:
The functionality in this method was already implemented [here](https://github.com/241209-JavaReactAWS/olga-nathanael-project1/blob/866baea7c20830975f18ac8b042e12b30a0ce0b5/backend/src/main/java/com/revature/happyfarmersmarket/interceptor/AuthenticationInterceptor.java#L53).

Reply:
Done